compiler/rustc_const_eval/src/transform/check_consts/check.rs

```diff
@@ -737,7 +742,7 @@ impl Visitor<'tcx> for Checker<'mir, 'tcx> {
                             return;
                         }
                     }
-                    self.check_op(ops::RawPtrDeref);
+                    self.check_op(ops::RawMutPtrDeref);
```

Comment:
I think we can safely bundle this with MutDeref. I see no reason to stabilize raw mut ptr deref separately from reference mut deref

Reply:
The reason I split it is because mutable references are unstable. If mutable pointer dereferencing is stabilized, doesn't that effectively open a back door?

Reply:
I meant to only use a single feature gate for mutable references and raw mutable pointers. This way, we will stabilize mutable references together with mutable pointers. What I meant to say is that I believe that once we stabilize mutable references, there is no reason not to also stabilize raw mutable pointers.

so basically use https://github.com/rust-lang/rust/blob/735c1339bd2a5e16dc0fde04f24930eda1bd049b/compiler/rustc_const_eval/src/transform/check_consts/ops.rs#L350-L369 here and adjust its message to also talk about mutable raw pointers.

Reply:
Ah, gotcha. I mentioned that in the tracking issue but edited it out afterwards. Should the feature gate's name stay the same or something else bikesheddable?